package.json

```diff
@@ -39,6 +39,7 @@
     "bookshelf": "^0.9.4",
     "co": "^4.6.0",
     "code": "^2.2.0",
+    "eslint": "^2.13.1",
```

Comment:
I was using node 6 and it's a peer dep from standard that isn't installed automatically, will do no harm here